[Tizen Studio 4.5] Release Notes (#1523)
Samsung/tizen-docs: docs/application/tizen-studio/release-notes/4-5-release-notes.md

# Tizen Studio 4.5 Release Notes

- Release Date: Oct 24, 2021

## IDE and Tools

### New Feature

- IDE

>   - Tizen 6.5 platform support has been added.

Reviewer: @manishtr would you also add `TIDL` support both on `IDE` and `CLI` into this `New Feature` section?

- CLI
  - New Resource Type Package has been added.

>   - New tool similar to tizen CLI has been added named tz which can be used to create,build,run dotnet,native and web projects.

Reviewer: ```suggestion
  - New Tizen CLI tool has been added named tz which can be used to create, build, run all the supported projects.
```

# Tizen Studio 4.5 Release Notes

- Release Date: Oct 24, 2021

## IDE and Tools

### New Feature

- IDE
  - Tizen 6.5 platform support has been added.
  - TIDL support has been added.
- CLI
  - New Resource Type Package has been added.
  - New Tizen CLI tool has been added named tz which can be used to create, build, run all the supported projects.
  - Permit to install feature has been added.
  - TIDL support has been added.
- VSCode
  - Web and Native support is added.
- Visual Studio(Windows)
  - Web and Native support is added.

### Deprecated
  - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.

### Known Issues

- Installer

>   - In macOS, Tizen Studio related file are marked as Malware, for which it is recommended to allow apps for anywhere option as described in https://www.imore.com/how-open-apps-anywhere-macos-catalina-and-mojave

Reviewer 2: ```suggestion
  - In macOS, Tizen Studio related file are marked as Malware, hence it is recommended to select **Anywhere** under allow apps as described in https://www.imore.com/how-open-apps-anywhere-macos-catalina-and-mojave
```

- CLI and SDB
  - Tizen Studio does not support the [Smart Development Bridge](../common-tools/smart-development-bridge.md) (SDB) bash auto-completion on Windows (it is available on Ubuntu and macOS).
- Dynamic Analyzer
  - When analyzing applications on commercial devices running Tizen 3.0, newly-released or after a firmware update, the following problems exist:
    - The Core Frequency information is not shown.
    - The screenshots on the scene transitions feature will not work.
  - When analyzing applications on the Tizen 4.0 emulator or reference device, the startup profiling information is not shown.
  - The UI Hierarchy viewer feature and startup profiling are not performed simultaneously.
  - The Dynamic Analyzer cannot show lifecycle information for Web applications.
  - Widget applications cannot be profiled with the Dynamic Analyzer. They are hidden in the application list on the toolbar for all Tizen platforms, except Tizen 2.3.2.
  - The Dynamic Analyzer sometimes gets stuck which is caused by an internal security checking program.
- VSCode
  - Hybrid Applications are not supported yet.
- Visual Studio

>   - Hybrid Applications are not supported yet.

Reviewer 2: ```suggestion
  - Hybrid applications are not supported yet.
```
